docs/how_to_guides/ingest_data/Derivation Strategy.md

```diff
@@ -0,0 +1,302 @@
+---
+id: derivation_strategies
+title: Schema Derivation Strategies
+
+---
+
+* Guzzle provides five schema derivation strategies. Using these strategies Guzzle will validate the data type and will also create an effective schema based on that data will be written in the target and reject table. The schema derivation strategies include:
+
+    1. Source
+    2. Inherit source data type and do not apply validation
+    3. Target
+    4. Inherit target data type and do not apply validation
+    5. Do not inherit schema from source or target
+
+* Users can select schema derivation strategy from **Inherit columns and datatype** property under **Additional settings** dropdown menu in the validation and transformation section.
+
+* In this article, we will discuss these strategies with examples
+
+## Source
+
+* If schema derivation strategy is **source** then, columns and data types will be inherited from the source files/table and, validation of data types will be performed based on source data types.
+* If the user does not specify the column name in the transformation and validation section then validation for that column will be performed and if the user specifies the source's column name in the transformation and validation section then the validation will be performed based on validate data type flag.
+* Additionally, an effective schema will be created using source columns and data types and columns and data types specified in the **validation and transformation** section of the ingestion job. And this effective schema will be used to create a table if an auto-create table flag is true in the target section and in the reject section.
```

Comment:
what about this ?

guzzle user: what is effective schema ? 🤔

effective schema will be created using 
1. source columns and there respective data types
2. columns and data type specify in validation and transformation